packages/react-three-rapier/src/Physics.tsx

```diff
@@ -66,6 +66,10 @@ export interface RigidBodyState {
 
 export type RigidBodyStateMap = Map<RigidBody["handle"], RigidBodyState>;
 
+export type WorldStepCallback = (world: World) => void;
```

Comment:
If we're looking to pass the world as a callback, maybe we should pass `WorldApi` for consistency with the other apis?

Reply:
I was thinking about this actually, and you are probably right!